src/pages/Transfer/Transfer.jsx

```diff
@@ -1,18 +1,65 @@
 import "./Transfer.scss";
+import Button from "../../components/Button/Button";
+const Transfer = ({
+  sendCurr = "GBP - British Pounds",
+  sendAmount = "£1000.00",
```

Comment:
sendAmount, recAmount and fee sould be just numbers. The money symbol (£,$) should be chosen based on sendCurr and recCurr